cluster/src/main/java/com/hartwig/pipeline/tertiary/virusbreakend/VirusBreakendOutput.java

```diff
@@ -0,0 +1,27 @@
+package com.hartwig.pipeline.tertiary.virusbreakend;
+
+import java.util.Optional;
+
+import com.hartwig.pipeline.StageOutput;
+import com.hartwig.pipeline.storage.GoogleStorageLocation;
+
+import org.immutables.value.Value;
+
+@Value.Immutable
+public interface VirusBreakendOutput extends StageOutput {
+
+    @Override
+    default String name() {
+        return VirusBreakend.NAMESPACE;
+    }
+
+    Optional<GoogleStorageLocation> maybeOutputDirectory();
```

Comment:
Does this get used by anything? If not fine to leave it out.

Reply:
Not used atm indeed so will take it out.